libgnucash/app-utils/gnc-ui-util.h

```diff
@@ -189,6 +189,13 @@ const char * gnc_get_reconcile_str (char reconciled_flag);
 const char * gnc_get_reconcile_valid_flags (void);
 const char * gnc_get_reconcile_flag_order (void);
 
+#define WASSOC 'w'
```

Comment:
Can this be `constexpr char WASSOC = 'w'`? I see the use in `gnc_get_association_str` and `gnc_get_association_flag_order` where `char` would fit in directly.

Reply:
`constexpr` is a C++11 feature. This is C code. So no, I don't think this can be used here. It's a valid improvement in a future C++ conversion though.